kotlin/workflow-ui-android/src/main/java/com/squareup/workflow/ui/ContainerHints.kt

```diff
@@ -30,7 +30,7 @@ class ContainerHints private constructor(
   private val map: Map<ContainerHintKey<*>, Any>
 ) {
   constructor(registry: ViewRegistry) :
-      this(mapOf<ContainerHintKey<*>, Any>(ViewRegistry to registry))
+      this(mapOf<ContainerHintKey<*>, Any>(ViewRegistry to (registry + defaultViewBindings)))
```

Comment:
@rjrjr I'm not a fan of doing this here, but I didn't see any other obvious place to put it. This is the one choke point where all ViewRegistries have to go through to get into the runtime.